client/src/modules/tasks/components/task/Task.tsx

```
@@ -30,79 +54,68 @@ const Task = (props) => {
   };
 
-  const changeTask = (e) => {
+  const changeTask = (e: FormEvent<HTMLFormElement>) => {
     e.preventDefault();
-    const change = {};
+    const change: TaskChanges = {};
     if (title.length > 0) change.taskName = title;
     if (details.length > 0) change.taskDetail = details;
-    if (deadline.length > 0) change.deadline = deadline;
-    if (online) dispatch(updateTask(change, token, props.id));
+    if (!!deadline) change.deadline = deadline;
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)